src/excmds.ts

```diff
@@ -144,8 +144,9 @@ export async function getNativeVersion(): Promise<void> {
  * That said, `bind gs fillinput [Tridactyl](https://addons.mozilla.org/en-US/firefox/addon/tridactyl-vim/) is my favourite add-on` could probably come in handy.
  */
 //#content
-export async function fillinput(...content: string[]) {
-    let inputToFill = DOM.getLastUsedInput()
+export async function fillinput(selector: string, ...content: string[]) {
```

Comment:
This is a breaking change if anyone used this.

Please document `fillinput NULL content to fill` and mention breaking change in changelog :)